lib/ProductOpener/Display.pm

```diff
@@ -9568,7 +9528,7 @@ JS
 	my @nutriments = ();
 	foreach my $nutriment (@{$nutriments_tables{$nutriment_table}}, @unknown_nutriments) {
 		push @nutriments, $nutriment;
-		if (($nutriment eq "fruits-vegetables-nuts-estimate-") and ($User{moderator})) {
+		if (($nutriment eq "fruits-vegetables-nuts-estimate-")) {
```

Comment:
Yes, I forgot to mention it above. It's useful info I think. It's not always correct as it depends on ingredients analysis, but I think it will be faster to make improvements to it if everyone can see it.